project/settings/defaults.py

```diff
@@ -81,6 +80,7 @@
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
+    "sslserver",
```

Comment:
Shouldn't this only be listed as a middleware when DEBUG = True? 

alongside this block on line 98
```python
if DEBUG:
    INSTALLED_APPS += ["debug_toolbar", "shells"]
    MIDDLEWARE_CLASSES += [
        "debug_toolbar.middleware.DebugToolbarMiddleware",
        "pyinstrument.middleware.ProfilerMiddleware",
    ]
```

Reply:
Yeah! I had ignored that there is more to the api than just development.

Reply:
I think it should be listed as an INSTALLED_APPS. I looked around there is nothing in the readme that shows it running as middleware, so I do not know how I can list there. All that said, I have been looking through the documentation, as I realized there is a big chunk of django that I have no idea how it works.